tests/legacy/functional/PaymentPlans/PaymentPlanEditSubscriptionWithPlanTest.php

```diff
@@ -57,6 +57,7 @@ public function testSucessfulCase()
 			)
 			->setExternalId('1000')
 			->setAmount(20000)
+            ->setSplitAmount(10000)
```

Comment:
indentation

Reply:
Fixed... Tabs vs 4 spaces. IDE uses 4 spaces, couldn't see the difference in my IDE.